pom.xml

```diff
@@ -53,6 +53,11 @@
             <artifactId>gson</artifactId>
             <version>2.8.5</version>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.retrofit2</groupId>
+            <artifactId>retrofit</artifactId>
+            <version>2.3.0</version>
+        </dependency>
```

Comment:
Added the retrofit dependency

Reply:
Is it used?  If not, this is technical debt.